website/docs/r/s3_bucket_policy.html.markdown

```diff
@@ -12,20 +12,37 @@ Attaches a policy to an S3 bucket resource.
 
 ## Example Usage
 
-### Using versioning
+### Creating a custom policy and adding to the bucket
```

Comment:
Since this is basic, I would either remove this line, or write basic. Thoughts 🤷‍♂️  😄 ?